### PR TITLE
Add user relation to message model

### PR DESCRIPTION
**Pull request by krekas**
When displaying messages to show who wrote message I need user relation in message model

### CODE REVIEW

File: src/Models/Message.php [hunk 1]
```diff
@@ -38,6 +39,16 @@ public function ticket(): BelongsTo
...
     }
 
+    /**
+     * Get User RelationShip
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function user(): BelongsTo
+    {
+        return $this->belongsTo(User::class);
```
Comment (ousid) [inline src/Models/Message.php:49]: Can you please follow the pattern like the rest of the methods? Like the `ticket()` method, and adding the foreign key to the config file under `messages` key will be appreciated!

If you can add few tests for the `user()` method under `MessageTest` file, will be perfect, for future changes.

**Review by ousid — CHANGES_REQUESTED** [review]

**Comment by krekas** [comment]
@ousid added foreign key to config, but sorry don't know how to add test when package doesn't have User model in Laravels default location.

**Comment by ousid** [comment]
Thanks, @krekas 

There's a default user model, for tests purposes, [here](https://github.com/coderflexx/laravel-ticket/blob/main/tests/Models/User.php), you can use `Coderflex\LaravelTicket\Tests\Models\User` namespace for the `User` model.

Here's an [example](https://github.com/coderflexx/laravel-ticket/blob/main/tests/Feature/TicketRelationsTest.php#L80)

**Comment by krekas** [comment]
@ousid the problem for me is in `Message` model with `User` relation. Let's say I have test like this
```php
it('can retrieve user from message', function () {
    $user = User::factory()->create();

    $ticket = Ticket::factory()->create();

    $tableName = config(
        'laravel_ticket.table_names.messages',
        'messages'
    );

    $message = Message::factory()
        ->create([
            $tableName['columns']['ticket_foreing_id'] => $ticket->id,
            $tableName['columns']['user_foreign_id']   => $user->id,
        ]);

    $this->assertDatabaseHas($tableName['table'], [
        $tableName['columns']['user_foreign_id'] => $user->id,
    ]);

    $this->assertEquals($ticket->messages->count(), 1);
    $this->assertEquals($message->user->id, $user->id);
});
```

And I get error `Error : Class "App\Models\User" not found` because well relation is belongs to user and here `User` model is from `App\Models\User`
```php
public function user(): BelongsTo
{
    $tableName = config('laravel_ticket.table_names.messages', 'messages');

    return $this->belongsTo(
        User::class,
        $tableName['columns']['user_foreign_id']
    );
}
```

**Comment by ousid** [comment]
Closing this PR, since it does not meet the requirements, and replaced with #16 

You may want to take a look at it.

Thanks for your contribution @krekas!

**Pull request closed without merging**